Annex60/Utilities/Math/IntegratorWithReset.mo

```diff
@@ -0,0 +1,174 @@
+within Annex60.Utilities.Math;
+block IntegratorWithReset "Output the integral of the input signal"
+  import Modelica.Blocks.Types.Init;
+
+  parameter Real k(unit="1")=1 "Integrator gain";
+
+  /* InitialState is the default, because it was the default in Modelica 2.2
+     and therefore this setting is backward compatible
+  */
+  parameter Modelica.Blocks.Types.Init initType=Modelica.Blocks.Types.Init.InitialState
+    "Type of initialization (1: no init, 2: steady state, 3,4: initial output)"
+    annotation(Evaluate=true,
+      Dialog(group="Initialization"));
+
+  parameter Real y_start=0 "Initial or guess value of output (= state)"
+    annotation (Dialog(group="Initialization"));
+
+  extends Modelica.Blocks.Interfaces.SISO(y(start=y_start));
```

Comment:
Having the `extends`-statement directly after `import`-statement would enhance readability